pkg/apis/core/types.go

```diff
@@ -2823,8 +2823,7 @@ type PodSpec struct {
 	// to run this pod.  If no RuntimeClass resource matches the named class, the pod will not be run.
 	// If unset or empty, the "legacy" RuntimeClass will be used, which is an implicit class with an
 	// empty definition that uses the default runtime handler.
-	// More info: https://git.k8s.io/enhancements/keps/sig-node/runtime-class.md
-	// This is a beta feature as of Kubernetes v1.14.
+	// More info: https://git.k8s.io/enhancements/keps/sig-node/585-runtime-class/README.md
```

Comment:
KEP moved as it was converted to the new format with the `kep.yaml` file. Are there more referecnes outside of this repo that we would not want to break? I can restore the file in enhancement repository with the redirect to the new README.